### PR TITLE
[feature-wip](global-dict) add thrift definition

### PR DESCRIPTION
**Pull request by yiguolei**
# Proposed changes

This is part of #9242

Add required thrift definition

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: gensrc/thrift/PaloInternalService.thrift [hunk 1]
```diff
@@ -274,8 +274,18 @@ struct TTxnParams {
...
 
-// ExecPlanFragment
+// Definition of global dict, global dict is used to accelerate query performance of low cardinality data
+struct TColumnDict {
+  1: optional Types.TPrimitiveType type
+  2: list<string> str_dict  // map one string to a integer, using offset as id
+}
 
+struct TGlobalDict {
+  1: optional map<i32, TColumnDict> dicts,  // map dict_id to column dict
```
Comment (BiteTheDDDDt) [inline gensrc/thrift/PaloInternalService.thrift:284]: Do we need a type as big as i32 here?

Comment (yiguolei) [reply gensrc/thrift/PaloInternalService.thrift:284]: Maybe need. Some times I find there are sql queries that have more than 10k lines, I am afraid there are many dicts. And the slot id has int32 type.

**Comment by morningman** [comment]
@yiguolei please edit the PR title, you can refer to http://doris.incubator.apache.org/zh-CN/community/how-to-contribute/commit-format-specification.html#commit-%E6%A0%BC%E5%BC%8F%E8%A7%84%E8%8C%83

**Comment by yiguolei** [comment]
> @yiguolei please edit the PR title, you can refer to http://doris.incubator.apache.org/zh-CN/community/how-to-contribute/commit-format-specification.html#commit-%E6%A0%BC%E5%BC%8F%E8%A7%84%E8%8C%83

already done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**